fairlearn/metrics/metrics_engine.py

```diff
@@ -11,20 +11,29 @@
 def metric_by_group(metric_function, y_true, y_pred, group_membership, sample_weight=None):
     """ Applies a metric to each subgroup of a set of data
 
-    :param metric_function
-    :type Function with signature (y_true, y_pred, sample_weight=None) which returns a scalar
+    Parameters
```

Comment:
Are you sure this is right? I'm 99% sure we use a different standard for the AML SDK.

Reply:
https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format
Pretty sure we want to stay with reST as opposed to numpy doc

Reply:
if we do want to switch we can use pyment

Reply:
Reading:
http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#module-sphinx.ext.autodoc
This is because I was using the Napoleon extension. Which suggests that the markup is not properly separated from the presentation :-(